docs/legacy/getting-started-apm-server.asciidoc

```diff
@@ -330,3 +445,31 @@ include::{libbeat-dir}/repositories.asciidoc[]
 
 // Shared docker
 include::{libbeat-dir}/shared-docker.asciidoc[]
+
+
+=== Fleet-managed APM Server
+
+// TODO:
+// These steps won't necessarily be on separate pages in the final version.
+
+==== Prerequisites
+
+Install {es} + {kib}
+
+==== Step 1: Set up Fleet
+
+Install {agent} and enable {fleet-server}.
```

Comment:
Are you adding this info and the other steps in a different PR?

Reply:
Nope! Added in this PR.